examples/benchmarks/memanto_vs_mem0/.env.example

```
@@ -0,0 +1,3 @@
+# Required: Get free key at https://console.moorcheh.ai/api-keys
+MEMANTO_API_KEY=***  # optional: Get key at https://app.mem0.ai
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)